go_modules/spec/fixtures/projects/unknown_revision_version/go.sum

```diff
@@ -0,0 +1,13 @@
+github.com/fatih/Color v1.7.0 h1:GPsoW9Vx62R0G4RLveWNf9++7g1zBbzTiyuqVCNHhyw=
```

Comment:
Uber-nit: this `go.sum` feels mighty invalid - I'm guessing it was copied from another fixture.

A couple of ideas:
* Since this project is invalid (e.g. `go mod tidy` will just exit without writing a `go.sum` file), just remove this file to align with that.
* Get this project into a working state with another version of `github.com/deislabs/oras`, or using a `require` directive to replace the invalid dependency. Store _that_ `go.sum`, then re-break the package.

I'd lean towards the (significantly faster) first.

Reply:
Good shout, removing `go.sum` 👍